modules/xmpp/xmpp.js

```diff
@@ -31,7 +31,12 @@ function createConnection(token, bosh = '/http-bind') {
         bosh += `${bosh.indexOf('?') === -1 ? '?' : '&'}token=${token}`;
     }
 
-    return new Strophe.Connection(bosh);
+    const conn = new Strophe.Connection(bosh);
+
+    // The default maxRetries is 5, which is too long.
+    conn.maxRetries = 3;
```

Comment:
@saghul Is `maxRetries` documented somewhere? I found it in Strophe's source code but I'm not able to find it in the documentation!

Reply:
I don't think it's documented. I saw it referenced in the changelog, some PRs and such.

Reply:
Yeah I saw the same! I'll merge the PR but probably we have to be careful with this!